docs/guide.md

```diff
@@ -88,6 +88,33 @@ environment variable:
 
 ```
 
+
+### Collecting Results
+
+Some API calls, such as searching for imagery and listing orders, return a
+varying, and potentially large, number of results. These API responses are
+paged. The SDK manages paging internally and the associated client commands
+return an asynchronous iterator over the results. These results can be
+converted to a JSON blob using the `collect` command. When the results
+represent GeoJSON features, the JSON blob is a GeoJSON FeatureCollection.
+Otherwise, the JSON blob is a list of the individual results.
+
+
+```python
+>>> import asyncio
+>>> from planet import collect, OrdersClient, Session
+>>>
+>>> async def main():
+...     async with Session() as sess:
+...         client = OrdersClient(sess)
+...         orders = client.list_orders()
+...         orders_list = collect(orders)
```

Comment:
@jreiberkyle @cholmes when I try to imagine that I'm reading that example without knowing any of the details, I wonder "Why doesn't list_orders return a list? Why do I need to call collect?" :phone:

In the Python standard library, I believe list_orders would be named iter_orders to communicate that it's an iterator that must be materialized into a list. And the canonical way of doing that would be to use Python's built in list function. Like `orders_list = list(iter_orders)`.

Perhaps we can emphasize the feature collection use case here instead of lists? Like `orders_collection = collect(orders)`?